From: Dan Bungert <daniel.bungert@canonical.com>
Date: Sun, 3 Dec 2023 11:24:19 -0400
Subject: /usr/include is a viable include dir

Bug-Debian: https://bugs.debian.org/999812
Forwarded: no
Last-Update: 2021-11-11

Upstream removed /usr/include from the default include locations, which breaks
the distutils test.  Revert of 6115fc52703baf07d4cfc629efb58866f0d16856.
---
 numpy/distutils/system_info.py | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/numpy/distutils/system_info.py b/numpy/distutils/system_info.py
index 6478548..eed09e6 100644
--- a/numpy/distutils/system_info.py
+++ b/numpy/distutils/system_info.py
@@ -330,7 +330,7 @@ else:
                                  '/opt/local/lib', '/sw/lib'], platform_bits)
     default_runtime_dirs = []
     default_include_dirs = ['/usr/local/include',
-                            '/opt/include',
+                            '/opt/include', '/usr/include',
                             # path of umfpack under macports
                             '/opt/local/include/ufsparse',
                             '/opt/local/include', '/sw/include',
@@ -339,7 +339,8 @@ else:
 
     default_x11_lib_dirs = libpaths(['/usr/X11R6/lib', '/usr/X11/lib',
                                      '/usr/lib'], platform_bits)
-    default_x11_include_dirs = ['/usr/X11R6/include', '/usr/X11/include']
+    default_x11_include_dirs = ['/usr/X11R6/include', '/usr/X11/include'
+                                '/usr/include']
 
     if os.path.exists('/usr/lib/X11'):
         globbed_x11_dir = glob('/usr/lib/*/libX11.so')
